scripts/ci/deploy-travis.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
This should probably be `#!/bin/bash` for consistency with all our other scripts